falcon/vendor/mimeparse/mimeparse.py

```diff
@@ -181,7 +181,7 @@ def best_match(supported, header):
         pos += 1
     weighted_matches.sort()
 
-    return weighted_matches[-1][0][0] and weighted_matches[-1][2] or ''
+    return weighted_matches[-1][2] if weighted_matches[-1][0][0] else ''
```

Comment:
Let us revert these changes. Sensible they may be, we have vendored this module from https://github.com/dbtsai/python-mimeparse, and do not want to make any modifications unless absolutely necessary.